File: drizzle/jruby/update/drizzle_update.rb
#! /usr/bin/jruby
# -*- coding: utf-8 -*-
#
#	drizzle_update.rb
#
#				Jul/16/2014
#
# ------------------------------------------------------------
include Java
import java.sql.DriverManager
import java.lang.System
#
require 'date'
#
load '/var/www/data_base/common/jruby_common/jruby_sql_manipulate.rb'
#
# ------------------------------------------------------------
puts "*** 開始 ***"
#
#
id_in = ARGV[0]
population_in = ARGV[1].to_i
#
puts id_in,population_in
#
str_connect = "jdbc:drizzle://localhost:4427/city"
#
conn = DriverManager.getConnection(str_connect,"scott", "tiger")

sql_update_proc(conn,id_in,population_in)
sql_read_proc(conn)

conn.close()
#
puts "*** 終了 ***"
#
# ------------------------------------------------------------

File: drizzle/python/update/drizzle_update.py
#! /usr/bin/python
# -*- coding: utf-8 -*-
#
#	drizzle_update.py
#
#						Jul/06/2012
#
import	sys
reload(sys)
sys.setdefaultencoding('utf-8')
import	string
#
sys.path.append ('/var/www/data_base/common/python_common')
from drizzle_manipulate import drizzle_update_proc
# ------------------------------------------------------------------
print	("*** 開始 ***")
id_in = sys.argv[1]
population_in = string.atoi (sys.argv[2])
print ("%s\t%d" % (id_in, population_in))
#
db_name = "city"
drizzle_update_proc (db_name,id_in,population_in)
#
print	("*** 終了 ***")
# ------------------------------------------------------------------

